README.md

```diff
@@ -1,52 +1,127 @@
 # OSRF Docker Images
-This repo hosts the Dockerfiles used to generate the official images for OSRF projects, including onbuild repositories on Docker Hub:
 
-##### Dockerfile CI [![Build Status](https://travis-ci.org/osrf/docker_images.svg?branch=master)](https://travis-ci.org/osrf/docker_images)
+This repo hosts Dockerfiles and scaffolding for images in the Official Library and OSRF Ornignation on Docker Hub.
+
+| CI | Status |
+|---|---|
+| Travis | [![Build Status](https://travis-ci.org/osrf/docker_images.svg?branch=master)](https://travis-ci.org/osrf/docker_images) |
+
+---
+
+## [Official Library](https://docs.docker.com/docker-hub/official_images)
+
+The following repos are hosted under the Official Library on Docker Hub. These images are tagged for production and general downstream use. If you’re targeting released versions of respective projects, then these are recommended.
+
+### [ROS ![Docker Pulls](https://img.shields.io/docker/pulls/_/ros.svg?label=pulls) ![Docker Stars](https://img.shields.io/docker/stars/_/ros.svg?label=stars)](https://hub.docker.com/_/ros)
+
+This repo contains images available for ROS, both ROS 1 and ROS 2. For more documentation on using these images, view the Docker Hub repo link above. Images are tagged by releases version, meta package, as well as code name for supported OS base images. For complete listing of tag, view the Repo Info link below.
```

Comment:
"tagged by releases version" is unclear to me. maybe ROS Distribution (code)name, or simply ROS Distribution